packages/utils/src/global.ts

```diff
@@ -44,3 +44,16 @@ export function getGlobalObject<T>(): T & SentryGlobal {
       : fallbackGlobalObject
   ) as T & SentryGlobal;
 }
+
+/**
+ * Returns a global singleton contained on the global `__SENTRY__` object.
+ *
+ * @param name name of the global singleton on __SENTRY__
+ * @param creator creation function
+ * @returns the singleton
+ */
```

Comment:
```suggestion
/**
 * Returns a global singleton contained in the global `__SENTRY__` object.
 *
 * If the singleton doesn't already exist in `__SENTRY__`, it will be created using the given factory function and added to the `__SENTRY__` object.
 *
 * @param name Name of the global singleton 
 * @param creator Factory function to create the singleton if it doesn't already exist on `__SENTRY__`
 * @param obj (Optional) The global object on which to look for `__SENTRY__`, if not `getGlobalObject`'s return value
 * @returns The singleton
 */
```